releases/release-1.30/release-notes/maps/pr-122293-map.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 122293
+releasenote:
+  text: 'kube-controller-manager: increased the global level for broadcaster''s logging
```

Comment:
Remove the extra ' in broadcaster's

Reply:
I think you can't have a single one here, otherwise, the checker reports a yaml formatting issue- is there a different way to escape the ' character?

Reply:
This lgtm! You can also try using the folded `|-` or double-quoted style `" "`